### PR TITLE
Don't use file storage

### PR DESCRIPTION
**Pull request by smoogipoo**
I have not tested this. Just a visual/compilation pass.

### CODE REVIEW

File: osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs [hunk 1]
```diff
@@ -95,88 +95,39 @@ public override double Calculate(Dictionary<string, double> categoryRatings = nu
...
+                finalsdpp = finalsdpp + (finalsd2 / stdevconst);
+
+                if (finalsdpp >= 1)
+                {
+                    finalsdpp = (-1) * (Log(finalsdpp, 0.000001));
+                }
             }
+
             double rawAim = Attributes.AimStrain + finalsdpp;
             // Always clear temp file after end sd workflow.
```
Comment (kornkaobat) [inline osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs]: Do we still need these lines? (Temp .db storage removal)

**Review by kornkaobat — CHANGES_REQUESTED** [review]
Maybe remove some unnecessary temp .db code I've written for my tempstorage file? (osuperformancecalculator.cs)

File: osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs (hunk 1 above)
Comment (smoogipoo) [reply osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs]: Ah I missed those, no they're not needed.

**Review by kornkaobat — APPROVED** [review]
Ok.

**Pull request merged**